### PR TITLE
Add note about secrets in README

### PR DESCRIPTION
**Pull request by Maximilianos**
Would have helped me just now :)

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ The doc site for [Fluxible](https://github.com/yahoo/fluxible).
 $ npm install
 ```
 
+There is also a /secrets.example.js file that you need to copy to /secrets.js with your GitHub API to allow the app to access the page content it needs from github.
```
Comment (jedireza) [inline README.md]: There is also a `/secrets.example.js` file that you need to copy to `/secrets.js` with your GitHub API to allow the app to access the page content it needs from GitHub.

- - - - -

I added ` around the file names and capitalized GitHub.

**Comment by jedireza** [comment]
Thanks. Just a small change I mentioned above.

**Comment by Maximilianos** [comment]
Nice :)

**Comment by jedireza** [comment]
:+1:

**Pull request merged**